scripts/api/entity/playerspaceship.lua

```diff
@@ -21,6 +21,14 @@ function PlayerSpaceship()
         transform = {rotation=random(0, 360)},
         callsign = {callsign=generateRandomCallSign()},
         scan_state = scan_state,
+        reactor = {},
```

Comment:
I don't like this. Part of the reason I added all these to playership templates, is that you can create fighter templates that don't need engineering. And thus those need to be able to remove a whole bunch of systems that aren't really for fighters.